Failing pr example

aborsu opened the pull request:
This branch should never be merged and is to be used as a failing pr example.

<!---
@huboard:{"order":4.0,"milestone_order":4,"custom_state":""}
-->

ghost commented:
<KGB>01fede614d999546f81223fb0c032db5788993bb

> doc(examples): Add tab example
> 
>   there is a  tab in  my commit message.

Use soft tabs!

efe793b23fa6426c48f2cafc02c8edd3eb4c1c9c

> doc(examples): Status line is well over 50 characters and for some reason it doesn't please us

Limit status to 50 characters

4dfe7692278c65528ef27638c797583232de800b

> Status line diverges from conventional changelog

Conform status to `<type>(<scope>): <subject>` pattern
type and scope cannot contain spaces or parenthesis

e352f326bc78d4effe703b5525081a4927a83eae

> doc(examples): Second line
> Is not empty

Separate status and body with an empty line.

6393daa589ec806deaa9611879697f0336b77eb8

> doc(examples): line too long
> 
> Я вас любил: любовь еще, быть может, В душе моей угасла не совсем; Но пусть она вас больше не тревожит; Я не хочу печалить вас ничем. Я вас любил безмолвно, безнадежно, То робостью, то ревностью томим; Я вас любил так искренно, так нежно, Как дай вам бог любимой быть другим.

Wrap body lines at 72 characters

Commit:
doc(examples): Add tab example

	there is a 	tab in	my commit message.

Commit:
doc(examples): Status line is well over 50 characters and for some reason it doesn't please us

Commit:
Status line diverges from conventional changelog

Commit:
doc(examples): Second line
Is not empty

Commit:
doc(examples): line too long

Я вас любил: любовь еще, быть может, В душе моей угасла не совсем; Но пусть она вас больше не тревожит; Я не хочу печалить вас ничем. Я вас любил безмолвно, безнадежно, То робостью, то ревностью томим; Я вас любил так искренно, так нежно, Как дай вам бог любимой быть другим.
